hazelcast/src/main/java/com/hazelcast/topic/impl/TopicService.java

```diff
@@ -172,7 +173,10 @@ public void publishMessage(String topicName, Object payload, boolean multithread
         }
     }
 
-    public String addMessageListener(String name, MessageListener listener, boolean localOnly) {
+    public @Nonnull
+    String addMessageListener(String name,
```

Comment:
Shouldn't `name` be `@Nonnull` as well? As far as I can see, the topic name is then used in various Map lookups internally which will fail with a NPE if the name is null. (And I just noticed that `removeMessageListener` annotates `name` as `@Nonnull`, so the absence of `@Nonnull` in this method is probably just an oversight.)

Reply:
Added `@Nonnull`. Can you recheck?